core/src/main/java/org/axonframework/config/DefaultEventProcessorRegistry.java

```diff
@@ -58,10 +60,14 @@
 public class DefaultEventProcessorRegistry implements EventProcessorRegistry {
```

Comment:
Isn't this a full-blown `EventProcessingConfiguration`? Conceptually, it sits really close to all the other `ModuleConfiguration` implementations.... It has the `initialize`, `start` and `shutdown` methods already...